Kconfig

```diff
@@ -17,6 +17,10 @@ config AOS_SM_VCHAN_RX_PATH
 	string "Path to the SM vchan"
 	default "/local/domain/1/tmp/vchan/sm_rx"
 
+config AOS_REBOOT_XEN_STORE_PATH
+	string "Path reboot"
+	default "/dev/xen/xenbus/control/user-reboot"
```

Comment:
It should be something similar to vchan path: "/local/domain/1/control/user-reboot". On linux domain it also possible to use relative path: "control/user-reboot". Could you check?

Reply:
Done, use the full path